packages/aws-cdk/lib/settings.ts

```diff
@@ -362,3 +361,20 @@ function expandHomeDir(x: string) {
   }
   return x;
 }
+
+/**
+ * Return all context value that are not an objecg with a _dontSaveContext member
+ */
+function stripTransientValues(obj: any) {
+  const ret: any = {};
+  for (const [key, value] of Object.entries(obj)) {
```

Comment:
`obj` might be `null` or `undefined`, in which case `Object.entries` will raise.

Reply:
It couldn't really, but I think you're telling me to shore up the input type definition.